src/World.gd

```diff
@@ -1,12 +1,28 @@
 extends Node2D
 
-onready var wonPop : PopupMenu = $CanvasLayer/WinPopup
+var wonPop : PopupMenu
 onready var floorTile : TextureRect = $Floor
 
+export var plataformsRequired := 2
+var plataformsPlaced := 0
 
 func _ready() -> void:
+	wonPop = $CanvasLayer/WinPopup
+	
 	floorTile.rect_size = get_viewport_rect().size
+	
+	Events.connect("boxPlaced", self, "onBoxPlaced")
+	Events.connect("gameWon", self, "onGameWon")
 
 
-func _on_Platform_gameWon() -> void:
-	wonPop.show()
+func onBoxPlaced() -> void:
+	print("Box placed")
```

Comment:
is the print needed? :)

Reply:
it's for testing will be removed when we fix the bug when removing the box from the target